lizmap/modules/lizmap/controllers/service.classic.php

```diff
@@ -1379,7 +1384,7 @@ public function DescribeFeatureType()
         // Get remote data
         list($data, $mime, $code) = lizmapProxy::getRemoteData($querystring);
 
-        if ($returnJson) {
+        if ($code/100 < 4 && $returnJson) {
```

Comment:
Please simplify the expression: 
```suggestion
        if ($code < 400 && $returnJson) {
```

Or maybe I'm missing something here....